packages/documentation/.storybook/addons/styles-switcher/StylesSwitcher.tsx

```diff
@@ -94,6 +96,49 @@ function StylesSwitcher() {
     );
   }, [preview, currentTheme, currentChannel]);
 
+  /**
+   * Sets the design system styles import SCSS file to the correct theme and channel file
+   */
+  useEffect(() => {
+    if (!stylesCodeBlocks) return;
+
+    const t = currentTheme.toLowerCase();
+    const c = currentChannel.toLowerCase();
+    const domain = "'@swisspost/design-system-styles/";
+
+    stylesCodeBlocks.forEach(stylesCodeBlock => {
+      let sourceArray = stylesCodeBlock.querySelectorAll('.token.string');
+      if (sourceArray.length) {
+        sourceArray.forEach((s, i) => {
```

Comment:
This three lines could be written as one.
Try it out if you like.

`Array.from(stylesCodeBlock.querySelectorAll('.token.string')).forEach((s, i) => {`

Reply:
I changed it to remove the if (sourceArray.length) but I kept the const sourceArray as I use it later in the code